CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_DefaultComplications.uc

```diff
@@ -58,11 +60,9 @@ function SpawnRescueMission(XComGameState NewGameState, XComGameState_ActivityCh
 	{
 		ChainTemplate = X2ActivityChainTemplate(TemplateManager.FindStrategyElementTemplate('ActivityChain_SupplyIntercept'));
 	}
-	
-	NewChainState = ChainTemplate.CreateInstanceFromTemplate(NewGameState);
-	NewChainState.FactionRef = ChainState.FactionRef;
-	NewChainState.PrimaryRegionRef = ChainState.PrimaryRegionRef;
 
+	ResContainer = XComGameState_ResourceContainer(NewGameState.CreateNewStateObject(class'XComGameState_ResourceContainer'));
+	
 	for (i = 0; i < ChainState.ChainObjectRefs.Length; i++)
```

Comment:
Are you sure that the intercepted chain's `ChainObjectRefs` refer to individual items? Somehow I really doubt that....

Reply:
I put the intercepted items inside there with my event listener return. I could have the ResContainer be created there though, instead of here.